api/base/serializers.py

```diff
@@ -44,6 +44,48 @@ def format_relationship_links(related_link=None, self_link=None, rel_meta=None,
     return ret
 
 
+class HideIfRetraction(ser.Field):
+    """
+    If node is retracted, this field will return None.
```

Comment:
Was this discussed? I'm not sure which is better: (1) return None for "hidden" fields or (2) actually hide the field, i.e. omit the output.